src/client/containers/BoardPageComponents/CandidateList/CandidateList.js

```diff
@@ -0,0 +1,65 @@
+import React, { useState, useEffect } from 'react';
+import { useCandidates } from '../../UseHooks/useCandidates';
+import { updateBallots } from './updateBallots';
+import { CardItemDecorator } from '../../../components/CandidateCard/CandidateCardItem.component';
+import { candidateCardSorting } from '../../../components/CandidateCard/CandidateCardSorting';
+import {
+  DragAndSortAdapter,
+  SortableItem,
+} from '../../DragAndSortAdapter/DragAndSortAdapter';
+import { onDragEnd } from '../../DragAndSortAdapter/OnDragEnd';
+import PropTypes from 'prop-types';
+
+export default function CandidateList({ userId, boardId }) {
+  const { candidates, setCandidates, error } = useCandidates(userId, boardId);
+  const [draggedInit, setDraggedInit] = useState(false);
+
+  function candidateTransform(candidate, index) {
+    return {
+      ...candidate,
+      rank: index + 1,
+    };
+  }
+
+  updateBallots(userId, boardId, candidates, draggedInit);
```

Comment:
Now it's working correctly. Is it right to just return updateBallots here like this?